### ISSUE TITLE
Might it not be a good idea...?

### ISSUE THREAD

**Issue by yb66** [aehlke/tag-it#398]
Hi @aehlke 

Might it not be a good idea to add some collaborators from the contributor list and get some help closing the (as of today) 164 issues and 71 pull requests? You don't seem interested in running this project any more but judging by the forks/issues/pulls, others are.

Regards,
iain

**Comment by aehlke** [comment]
I agree it's necessary to sustain a project - I've had trouble seeing who stands out as a good candidate to add as a collaborator, but I haven't put work into it yet either (something I should have done while it was under active maintenance).

I would like to revisit this when I have some downtime soon. Appreciate the nudge.

**Comment by yb66** [comment]
@aehlke No probs, I've been clearing up some pull requests that have been lingering for years (oops!) so I feel your pain!
